cmd/internal/cli/overlay_create.go

```diff
@@ -0,0 +1,50 @@
+package cli
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/sylabs/singularity/docs"
+	"github.com/sylabs/singularity/internal/app/singularity"
+	"github.com/sylabs/singularity/pkg/cmdline"
+	"github.com/sylabs/singularity/pkg/sylog"
+)
+
+var (
+	overlaySize int
+	overlayDirs []string
+)
+
+// -s|--size
+var overlaySizeFlag = cmdline.Flag{
+	ID:           "overlaySizeFlag",
+	Value:        &overlaySize,
+	DefaultValue: 64,
+	Name:         "size",
+	ShortHand:    "s",
+	Usage:        "size of the EXT3 writable overlay in MB",
```

Comment:
Presumably this is MiB? If so can we put MiB here as we have had some confusion previously with powers of 10 MB vs powers of 2 MiB in some code.